tests/services/test_jwt_service.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from app.core.exceptions.token import DecodeTokenException
+from app.services.jwt_service import JwtService
+
+class TestJwtService(unittest.IsolatedAsyncioTestCase):
+    async def asyncSetUp(self):
+        self.jwt_service = JwtService()
+
+    @patch('app.services.jwt_service.TokenHelper')
+    async def test_verify_token_valid(self, mock_token_helper):
+        mock_token_helper.decode.return_value = {"sub": "refresh", "id": 1, "onyen": "test_user"}
+        await self.jwt_service.verify_token("valid_token")
+        mock_token_helper.decode.assert_called_once_with(token="valid_token")
```

Comment:
I don't think this is testing anything useful tbh. And it will just give us coverage without utility, so we won't think to actually test it later.